recipes/nanoeigenpy/build.bat

```diff
@@ -0,0 +1,29 @@
+setlocal EnableDelayedExpansion
+
+rd /q /s build
+
+mkdir build
+cd build
+
+set "CC=clang-cl.exe"
+set "CXX=clang-cl.exe"
+set "CL=/MP"
```

Comment:
This only add warning : ` clang-cl: warning: argument unused during compilation: '/MP' [-Wunused-command-line-argument]`